### PR TITLE
Update installation guide

### PR DESCRIPTION
**Pull request by Splines**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Update documentation (for installation):
   - Replace `docker compose` with `docker compose` since `docker compose` will be deprecated soon:
     > From the end of June 2023 Compose V1 won’t be supported anymore and will be removed from all Docker Desktop versions.
from [here](https://docs.docker.com/compose/compose-file/)
   - Update repo url
   - Beautify Readme a bit (e.g. MaMpf logo header)

* **Please check if the PR fulfills these requirements**
  - [ ] E2E Tests for the changes have been added  via Cypress
  - [ ] Meaningful rspec tests have been added
  - [X] Docs have been added / updated 

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
-## Installation (with docker-compose)
+## Installation (with docker compose)
 
-To simply try out mampf you can use `docker-compose` ([needs docker](https://docs.docker.com/engine/install/ubuntu/) && `apt install docker-compose`). Simply clone the mampf repository and run docker-compose by executing
+To simply try out mampf you can use `docker compose` (needs [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/)). Simply clone the MaMpf repository and run docker compose by executing
 ```
-$ git clone -b main --recursive https://github.com/MaMpf-HD/mampf.git
-
+$ git clone -b main --recursive git@github.com:MaMpf-HD/mampf.git
```
Comment (christian-heusel) [inline INSTALL.md:5]: I dont know whether this is an actual improvement, as git over http requires no configuration and is therefore easily doable. Git over ssh has the advantage that later development is more easily doable without changing the git upstream url.

File: INSTALL.md [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
 
-NOTE: Please make sure to clone recursively as the pdf compression feature is in an extra repository.
-If you have an already checked out version simply run:
+NOTE: Please make sure to clone recursively as the pdf compression feature is in a separate repository.
 
-```sh
+If you have an already checked out version simply run:
+```
 git submodule init
 git submodule update
```
Comment (christian-heusel) [inline INSTALL.md:15]: This could also be simplified:
```suggestion
git submodule update --init
```

File: INSTALL.md [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
 You now have the following things ready:
-* The MaMpf server on <a href="http://localhost:3000/" target="_blank">localhost:3000</a>
-* The mailcatcher service on <a href="http://localhost:1080/" target="_blank">localhost:1080</a>
-* The webinterface for ApacheSolr on <a href="http://localhost:8983/" target="_blank">localhost:8983</a>
+* The MaMpf server on [localhost:3000](http://localhost:3000/)
+* The mailcatcher service on [localhost:1080](http://localhost:1080/)
+* The webinterface for ApacheSolr on [localhost:8983](http://localhost:8983/)
 * A test mailserver instance on Ports 1025, 10143, 10993
-* A webpacker dev server on localhost:3035
+* A webpacker dev server on [localhost:3035](http://localhost:3035/)
```
Comment (christian-heusel) [inline INSTALL.md:23]: Please undo the changes to this section, automatically having new tabs opened via `target="_blank"` is pretty handy 😊

File: README.md [hunk 4]
```diff
@@ -51,13 +56,14 @@ MaMpf is implemented in Ruby on Rails.
 * Test suite: rspec, cypress
 * support for I18n
 
-## Installation (with docker-compose)
+## 💻 Installation (with docker compose)
+
+To easily try out MaMpf you can use docker compose. Clone the MaMpf repository and run docker compose:
```
Comment (christian-heusel) [inline README.md:61]:
```suggestion
To easily try out MaMpf you can use `docker compose`. Clone the MaMpf repository and run `docker compose`:
```

**Review by christian-heusel — CHANGES_REQUESTED** [review]
Overall looks good, needs some minor adjustments ✅ 🔥

File: INSTALL.md (hunk 3 above)
Comment (Splines) [reply INSTALL.md:23]: However, GitHub markdown does not reflect this, right?

File: INSTALL.md (hunk 1 above)
Comment (Splines) [reply INSTALL.md:5]: True, but right now we have ssh in the Readme and http in the Install markdown. Will go with http as no configuration is required ;)

**Review by christian-heusel — APPROVED** [review]

**Comment by Splines** [comment]
changed base branch from `main` to `mampf-next` to really stick to our own guidelines ;)
